### DISCUSSION TITLE
【讨论】The Silent Void 这标题，感觉用中文很难翻译啊

### DISCUSSION THREAD

**Discussion by fyzzy1943** [yikeke/tao-of-programming#3] [Q&A]
看到void，可以感受到虚无；但是看到虚无，只是明白了虚无

**Comment by yikeke** [comment]
嗯，这里肯定有双关，甚至三关（虚空（强调空间），虚无（强调内心感受），以及代码中的 void）。确实难翻，可以再讨论下～

**Comment by Johnsonhtao** [comment]
似静非静 似空非空
从表面看没有，当你能看出来时。它的确是存在的，很安静待在在那里，等你来发现的编程之道。
